analyzer/java/com/engflow/bazel/invocation/analyzer/dataproviders/CachingAndExecutionMetrics.java

```diff
@@ -201,7 +214,10 @@ public String getSummary() {
     sb.append(
         summarize(
             width, actionsWithoutRcRemotelyExecuted, getActionsWithoutRc(), "of all cache skips"));
-    sb.append("\n        Not reported (e.g. internal): ");
+    sb.append("\n        Internal (not exhaustive):    ");
```

Comment:
I think the "not exhaustive" remark raises more questions than it answers. I understand that its intention is to clarify, but the user needs to be aware of the nuance of identifying internal actions to understand why exactly this is not exhaustive.

How about something less precise but also less confusing (IMO):

```suggestion
    sb.append("\n        Internal (estimated):         ");
```

Reply:
The thing is, I know we are mostly underreporting. For example, one of the profiles I'm working with has a discrepancy of about 50%. (should report 4xxx, actually does report 2xxx)

Note that the summaries in the data providers are not shown in suggestions. They are only shown
in the webUI: by switching to "view data" mode
in the CLI: by specifying an additional param, `--mode` which includes `used_data` or `all_data`

That is to say, this data is less often viewed (and rather by "powerusers") and communicate as being more "raw".

Reply:
If our confidence is so low (50%), is it valuable to show any estimation?

Reply:
It's data, which may be used for suggestions.
For example, if I want to exclude internal actions from a suggestion, it's better to at least be able to exclude some of them, even if I don't catch all of them.
Concrete example: I don't want any suggestions that tell me to investigate symlinks, but there are tons of those. So if we can at least exclude those, it's something.